docs/core/tools/dotnet-new-sdk-templates.md

```diff
@@ -90,15 +92,15 @@ Each template may have additional options available. The core templates have the
 
 - **`--use-program-main`**
 
-  If specified, an explicit `Program` class and `Main` method will be used instead of top-level statements. Available since .NET SDK 6.0.300. Default value: `false`.
+  If specified, an explicit `Program` class and `Main` method will be used instead of top-level statements. Available since .NET SDK 6.0.300. Default value: `false`. Available only in C# language.
```

Comment:
```suggestion
If specified, an explicit `Program` class and `Main` method will be used instead of top-level statements. Available since .NET SDK 6.0.300. Default value: `false`. Available only for C#.
```